Naïve Modal component

sapegin opened the pull request.

okonet commented on src/exercises/7-Naive_Modal/Propmpt.js:
```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import Input from '../../components/primitives/Input';
+import Button from '../../components/primitives/Button';
+import Heading from '../../components/primitives/Heading';
+import Stack from '../../components/layout/Stack';
```
Lets not use any of those and just HTML since that's what we expect from participants.

sapegin replied:
Didn't we agreed on introducing primitives with a short exercise _before_ this one?

okonet commented on src/exercises/6-Dialog/Dialog.js:
```diff
@@ -0,0 +1,46 @@
+					/>
+				</label>
+				<div>
+					<button type="submit" variant="primary">
```
Variant? Let’s remove

sapegin commented:
I did some cleanup, added more text and added templates. Going to merge now, so I can continue with the next exercise. Feel free to comment though — I'll fix in the next pull request.
